### PR TITLE
Grammar stuff

### PR DESCRIPTION
**Pull request by allisonchihak**

### CODE REVIEW

File: mews-api.md [hunk 1]
```diff
@@ -517,31 +517,31 @@ This is example of a _successful_ response. In case an error occurred, the respo
 
 ### Process Group
 
-\[`async`\] This operation allows channel manager to push reservation group \(i.e. _booking_\) to Mews. This option allows creations, modifications and \(partial\) cancellations. Mews will process and confirm the booking asynchronously.
+\[`async`\] This operation allows the channel manager to push reservation groups \(i.e. _bookings_\) to Mews. This option allows creations, modifications and \(partial\) cancellations. Mews will process and confirm the booking asynchronously.
```
Comment (ondrahermanek) [inline mews-api.md]: Debatable, it allows to push just 1 group per call.

File: mews-api.md [hunk 2]
```diff
@@ -517,31 +517,31 @@ This is example of a _successful_ response. In case an error occurred, the respo
...
 
 #### Request `[PlatformAddress]/api/channelManager/v1/processGroup`
 
-The example shows a valid group definition with 2 space reservations + cancellation of 3rd space reservation. First `reservation` definition shows all details, second `reservation` definition shows the minimal required details for creation / modification of a `reservation`. The 3rd `reservation` definition shows the partial cancellation - cancelling the 3rd space reservation.
+The example shows a valid group definition with 2 space reservations + cancellation of a 3rd space reservation. First `reservation` definition shows all details, second `reservation` definition shows the minimal required details for creation / modification of a `reservation`. The 3rd `reservation` definition shows the partial cancellation - cancelling the 3rd space reservation.
```
Comment (ondrahermanek) [inline mews-api.md]: would `the` be better?

File: mews-api.md (hunk 1 above)
Comment (allisonchihak) [reply mews-api.md]: Agree, will re-submit with:
a reservation group \(i.e. _booking_)\

File: mews-api.md (hunk 2 above)
Comment (allisonchihak) [reply mews-api.md]: Yes 😄

**Pull request merged**